cmd/shell.go

```diff
@@ -335,3 +364,15 @@ func LoadCert(path string) (cert []byte, err error) {
 	cert = block.Bytes
 	return
 }
+
+func DebugServer(addr string) {
+	if addr == "" {
+		log.Fatalf("unable to boot debug server because no address was given for it. Set debugAddr.")
```

Comment:
Yeah, debugging should always be enabled. It doesn't cost anything to make available (it only adds over head when being called) and when you need it is when things are already going bad. For instance, when a production machine is GC spiraling for unknown reasons.